Podfile.lock

```diff
@@ -11,37 +11,37 @@ PODS:
     - Adyen/Card
     - Adyen/Core
   - Adyen3DS2 (2.1.0-rc.6)
-  - Braintree/Card (4.37.1):
```

Comment:
Everything seems to be working as expected. So it appears there is no issue.

Though this is what I have been used to so far where cocoa pods will always 100% update these all the time any time pod install or pod update is ran since it is required to run these each time without it refusing to compile due to an outdated podfile